### ISSUE TITLE
Setting minimumSystemVersion for macOS not possible

### ISSUE THREAD

**Issue by acarlsen** [JetBrains/compose-multiplatform#4583]
**Describe the bug**
As documented here we should be able to specify minimumSystemVersion for MacOS builds:
https://github.com/JetBrains/compose-multiplatform/tree/master/tutorials/Native_distributions_and_local_execution).
This is not possible and I get an "Unresolved reference: minimumSystemVersion" error when building projects.

I would like to set minimum version to 12.0.0. I can add custom plist key with extraKeysRawXml, but then I get an error when trying to upload to testflight:

```
Asset validation failed (90264)
Invalid product archive. The lowest minimum system version in the product definition property list, 10.13, must equal the LSMinimumSystemVersion value, 12.0.0. (ID: 4db7c056-bd2c-413a-b469-f0cc5022e2e1)
```

It seems the gradle plugin code, if we were allowed to specify minimumSystemVersion, would actully set the info.plist correctly and also set the product-def.plist correctly. As seen on lines 499-514 here: 
https://github.com/JetBrains/compose-multiplatform/blob/master/gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractJPackageTask.kt

**Affected platforms**
- macOS

**Versions**
- Kotlin version: 1.9.22
- Compose Multiplatform version: 1.6.1 
- OS version(s)* (required for Desktop and iOS issues): All
- OS architecture (x86 or arm64): x86/arm64
- Device (model or simulator for iOS issues): N/A
- JDK (for desktop issues): jbrsdk-21.0.2-osx-aarch64-b346.3

**To Reproduce**
Steps and/or the code snippet to reproduce the behavior:
1. Add compose.desktop.application.nativeDistributions.macOS.minimumSystemVersion key to build.gradle.kts file for jvm project
2. Sync project
4. See error (error also visible in IDE as compile error)

**Expected behavior**
That we can set minimumSystemVersion as documented.

**Comment by terrakok** [comment]
The minimumSystemVersion property is available since compose 1.6.10 only:
https://github.com/JetBrains/compose-multiplatform/commit/e1aff758c0d6c9b2547542a934b643cf25c8f1b1

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4583">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
